test/Distributed/SIL/distributed_actor_default_init_sil_8.swift

```diff
@@ -0,0 +1,102 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-swift-frontend-emit-module -emit-module-path %t/FakeDistributedActorSystems.swiftmodule -module-name FakeDistributedActorSystems -disable-availability-checking %S/../Inputs/FakeDistributedActorSystems.swift
+// RUN: %target-swift-frontend -module-name default_deinit -primary-file %s -emit-sil -verify -disable-availability-checking -I %t | %FileCheck %s --enable-var-scope --dump-input=fail
+// REQUIRES: concurrency
+// REQUIRES: distributed
+
+/// The convention in this test is that the Swift declaration comes before its FileCheck lines.
+
+import Distributed
+import FakeDistributedActorSystems
+
+/// This actor system is a class, therefore SIL is slightly different
+typealias DefaultDistributedActorSystem = FakeRoundtripActorSystem
+
+// ==== ----------------------------------------------------------------------------------------------------------------
+
+class SomeClass {}
+
+enum Err : Error {
+  case blah
+}
+
+func getSomeClass() throws -> SomeClass { throw Err.blah }
+func getSystem() throws -> FakeRoundtripActorSystem { throw Err.blah }
+
+distributed actor MyDistActor {
+  var someField: SomeClass
+
+  init() throws {
+    do {
+      actorSystem = try getSystem()
+    } catch {
+      actorSystem = FakeRoundtripActorSystem()
+    }
+    someField = try getSomeClass()
+  }
+
```

Comment:
This adds a test for a class based actor system which was a thing we didn't cover well, but in the real world most systems will be; This is the same as the other sil_n tests but for a class.